script.js

```diff
@@ -20,7 +20,7 @@ function appendCanvasesToHTML() {
 
 document.getElementById('file-input').onchange = function (event) {
     const fr = new FileReader();
-    fr.onload = setupNifti;
+    fr.onload = (event) => setupNifti(event.target.result);
```

Comment:
Had to fix this, since you change the setupNifti to take a file instead of the fileReader event

Reply:
Oh! I push this commit for #8 into master instead of the remote-data branch by mistake, sorry and thanks for catching that bug.

Reply:
Yeah I was surprise to see those changes bundled in this pr. All good 👌